.gitignore

```diff
@@ -68,3 +68,4 @@ cmake-build-debug/
 build.ninja
 rules.ninja
 .ninja_*
+.DS_Store
```

Comment:
I thought we already ignore `/**/.DS_Store`. Is this not working for `.DS_Store` in the root folder?
In either case, please make a new PR for this change, it is unrelated to the rest of this PR.